feature/metis/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/ui/SinglePageConversationBody.kt

```diff
@@ -38,10 +56,36 @@ internal fun SinglePageConversationBody(
         }
     }
 
+    var canCreateChannel by rememberSaveable { mutableStateOf(false) }
+    val coroutineScope = rememberCoroutineScope()
+
+    LaunchedEffect(courseId) {
+        coroutineScope.launch {
```

Comment:
Why do you launch in a LaunchedEffect? A launched effect already supports coroutines.